driver/ppm_events_public.h

```diff
@@ -765,7 +765,9 @@ enum ppm_event_type {
 	PPME_CONTAINER_JSON_X = 273,
 	PPME_SYSCALL_SETSID_E = 274,
 	PPME_SYSCALL_SETSID_X = 275,
-	PPM_EVENT_MAX = 276
+	PPM_EVENT_MAX = 276,
+	PPME_SYSCALL_TIME_E = 277,
```

Comment:
That's not correct, PPM_EVENT_MAX as the name says should be the last entry here.